cicd/gitops/configuring-sso-for-argo-cd-using-keycloak.adoc

```diff
@@ -0,0 +1,27 @@
+[id="configuring-sso-for-argo-cd-using-keycloak"]
+= Configuring SSO for Argo CD using Keycloak
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: configuring-sso-for-argo-cd-using-keycloak
+
+toc::[]
+
+After the {gitops-title} Operator is installed, Argo CD automatically creates a user with `admin` permissions. To manage multiple users, Argo CD allows cluster administrators to configure SSO.
```

Comment:
```suggestion
After the {gitops-title} Operator is installed, Argo CD automatically creates a user with `admin` permissions. To manage multiple users, cluster administrators can use Argo CD to configure Single Sign-On (SSO).
```
Per the IBM SG - "Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can...""